src/Visitor/VisitorStrategyAbstract.php

```diff
@@ -21,4 +30,55 @@ protected function getVisitor()
     {
         return $this->visitor;
     }
+
+    /**
+     * @return ConfigManager
+     */
+    protected function getConfigManager()
```

Comment:
for all this MR, check all return typings and var typing pease :)

Reply:
if it is ok with the php version you want to supporrt

Reply:
unfortunately, no